src/Qir/Runtime/lib/Simulators/FullstateSimulator.cpp

```diff
@@ -59,14 +59,14 @@ QUANTUM_SIMULATOR LoadQuantumSimulator()
     return handle;
 }
 
-bool UnloadQuantumSimulator(QUANTUM_SIMULATOR handle)
-{
-#ifdef _WIN32
-    return ::FreeLibrary(handle);
-#else // not _WIN32
-    return ::dlclose(handle);
-#endif
-}
+// bool UnloadQuantumSimulator(QUANTUM_SIMULATOR handle)

```

Comment:
I think it would be fine to remove this rather than leave it commented out. I don't foresee us ever really supporting this.

Reply:
Removing.